ors-api/src/main/java/org/heigit/ors/api/responses/snapping/SnappingResponse.java

```diff
@@ -0,0 +1,12 @@
+package org.heigit.ors.api.responses.snapping;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.heigit.ors.snapping.SnappingResult;
+public class SnappingResponse {
+    @JsonIgnore
+    private final SnappingResult result;
+
+    public SnappingResponse(SnappingResult result) {
+        this.result = result;
+    }
```

Comment:
Is this already sufficient, in terms of swagger docs?

Reply:
Yes, the Response object types are built from the locations array and JSONLocation Object
![grafik](https://github.com/GIScience/openrouteservice/assets/23240110/bed4fd3b-5fec-4c3d-b974-86bcfca2dcac)